scripts/setup

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+# this shell script is responsible for:
+# - making sure your environment is setup, and
+# - setting up a database environment locally for you to use
+set -e
+
+usage() {
+  echo "usage: $0 [-h|--help]"
+  echo ""
+  echo "[-h|--help]      print this help text"
+}
+
+# script variables and defaults
+source $(dirname $0)/init.sh
+
+# the list of tools required to be installed to interact with this repo
+# this is meant to verify what is also in the ./README.md#environment-setup section
+ROOT_DIR="$(cd "$(dirname "$0")/.."; pwd)"
```

Comment:
this is set in `init.sh` - feel free to remove / cleanup

Reply:
Done